Formula/mmctl.rb

```diff
@@ -23,20 +23,14 @@ class Mmctl < Formula
   depends_on "go" => :build
 
   def install
-    ENV["GOBIN"] = buildpath/bin
-    ENV["ADVANCED_VET"] = "FALSE"
-    ENV["BUILD_HASH"] = Utils.git_head
-    ENV["BUILD_VERSION"] = version.to_s
-    (buildpath/"src/github.com/mattermost/mmctl").install buildpath.children
-    cd "src/github.com/mattermost/mmctl" do
-      system "make", "install"
+    ldflags = "-s -w -X github.com/mattermost/mmctl/commands.BuildHash=#{Utils.git_head}"
+    system "go", "build", *std_go_args(ldflags: ldflags), "-mod=vendor"
```

Comment:
in addition to removing `make`'s indirection, this avoids `gofmt` errors with go 1.17

Reply:
👍